src/main/java/com/unascribed/drogtor/DrogtorMod.java

```diff
@@ -73,6 +75,29 @@ public void onInitialize() {
 						informNamecard(c.getSource().getPlayer());
 						return 1;
 					}));
+			dispatcher.register(LiteralArgumentBuilder.<ServerCommandSource>literal("bio")
+					.then(RequiredArgumentBuilder.<ServerCommandSource, String>argument("bio", StringArgumentType.greedyString())
+							.executes((c) -> {
+								// replace § just in case
+								String bio = c.getArgument("bio", String.class).replace("§", "");
+								bio = Pattern.compile("\\\\.").matcher(bio).replaceAll(res -> {
```

Comment:
Pattern should be cached in a private static final field

Reply:
thought you wouldn't want fields in your pristine empty class, heh